### PR TITLE
Ember 2.0 attrs support

### PR DESCRIPTION
**Pull request by miguelmota**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-i18n",
-  "version": "3.1.1",
+  "version": "3.1.2",
```
Comment (jamesarosen) [inline bower.json]: Would you mind resubmitting this without the version bump?

File: lib/i18n.js [hunk 2]
```diff
@@ -126,18 +130,18 @@
         });
       },
 
-      _scheduleObservers: Ember.on('init', function() {
+      _scheduleObservers: function() {
```
Comment (jamesarosen) [inline lib/i18n.js]: Not all apps have prototype extensions enabled. The official policy is that addons *shouldn't* rely on them.

Comment (locks) [inline lib/i18n.js]: The official policy is that addons have prototypes turned off by default actually (starting with 0.2.x?), so this is good advice

**Comment by locks** [comment]
@miguelmota as a rule of thumb you shouldn't bump versions in PRs, since the maintainer of the library usually wants to control the versioning.

The 1.10 branch is failing the tests because the HTMLBars compiler isn't required it seems.

**Comment by jamesarosen** [comment]
v4.0 will support Ember 2.0. The v3.x branch never will.

**Pull request closed without merging**